mixins/StoreMixin.js

```diff
@@ -76,9 +76,16 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
```

Comment:
I ran into the same thought process when debugging this guy too. We probably shouldn't be changing the data structure